.github/workflows/ci.yml

```diff
@@ -0,0 +1,61 @@
+name: Scala Merge CI
+
+on:
+  push:
+    branches: ['2.*.x']
+  pull_request:          ## TODO remove this line before merging
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  build_and_test:
+    name: Windows
+    runs-on: windows-latest
+    strategy:
+      fail-fast: false
+    steps:
+      - run: git config --global core.autocrlf false
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      # Note that we don't use olafurpg/setup-scala; it wouldn't buy us anything
+      # over setup-java. (We don't want csbt or xsbt; we prefer the standard
+      # sbt launch script, which comes preinstalled on Windows (and Ubuntu).)
+      - name: Setup Java
+        uses: actions/setup-java@v2
+        with:
+          distribution: adopt
+          java-version: 8
+
+      - name: Cache
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.sbt
+            ~/.ivy2/cache
+            ~/.cache/coursier
+          key: ${{ runner.os }}-sbt-cache-v2-${{ hashFiles('**/*.sbt') }}-${{ hashFiles('project/build.properties') }}
+
+      - name: Setup
+        run: |
+          source scripts/common
+          java -version
+          javac -version
+          generateRepositoriesConfig
+          # Pass these environment vars to subsequent steps
+          echo "SBT=sbt -Dsbt.override.build.repos=true -Dsbt.repository.config=${sbtRepositoryConfig}" >> $GITHUB_ENV
```

Comment:
we don't do `scripts/common` / `generateRepositoriesConfig` / `sbt.override.build.repos` in the travis build, why here?

https://github.com/scala/scala/blob/52cdad95982e0bf05d00a0caaaba7d6d2b490a2d/.travis.yml#L43-L46

Reply:
Probably no reason except that we were copying `scripts/jobs/integrate/windows` rather than copying `.travis.yml`, which have probably diverged from each other accidentally. I agree we should try for the simpler version.

Reply:
Cool! The pr testing in travis doesn't use anything from `scripts`, which is what should be our long-term goal.